pulsar-client-cpp/tests/BasicEndToEndTest.cc

```diff
@@ -1756,7 +1756,7 @@ TEST(BasicEndToEndTest, testPatternTopicsConsumerInvalid) {
 // verify PatternMultiTopicsConsumer subscribed matched topics,
 // and only receive messages from matched topics.
 TEST(BasicEndToEndTest, testPatternMultiTopicsConsumerPubSub) {
-    Client client(lookupUrl);
+    Client client(adminUrl);
```

Comment:
Ideally we should have the test fetching the lists of topic using both HTTP and protobuf.